posthog/api/feature_flag.py

```diff
@@ -106,3 +109,9 @@ def get_queryset(self) -> QuerySet:
         if self.action == "list":
             queryset = queryset.filter(deleted=False)
         return queryset.order_by("-created_at")
+
+    @action(methods=["GET"], detail=False)
+    def user_status(self, request: request.Request, **kwargs):
+        distinct_id = request.GET.get("distinct_id", None)
+        flags = get_active_feature_flags(self.team, distinct_id)
+        return Response({"distinct_id": distinct_id, "flags_enabled": flags})
```

Comment:
Not a deal breaker, but I think it's worth using a serializer here to handle this logic (and reading the `GET` params too), it's cleaner and standardized. If you do keep it this way, probably worth switching to `JsonResponse` instead.

Reply:
Hmm. I chose `Response` just so we could have the nicer looking Django API like on ~~all~~ most other endpoints:

![image](https://user-images.githubusercontent.com/53387/123937204-37d59c80-d996-11eb-9265-976afc1c9974.png)

... yet it should still work if you ask for the json programmatically:

![image](https://user-images.githubusercontent.com/53387/123937436-74a19380-d996-11eb-935e-bdfde9e6c79e.png)

As for moving this to a serializer. I'm willing to do it if it makes sense. Yet with keeping things simple, a 3-line function body seems to have less overhead than making another class. Had it been an normal API endpoint that requires some serialization and sanitization, I'd definitely gone for it, but now for a custom JSON endpoint it feels like overkill. I might be wrong though --> I don't have muscle memory for django best practices, yet :).

Reply:
Yeah this seems fine. Also checked, and the `Content-Type` is properly set, so 👍 

<img width="962" alt="" src="https://user-images.githubusercontent.com/5864173/124066015-2e047580-da38-11eb-8e3c-53c63dabe047.png">